src/components/App/styles.css

```diff
@@ -114,150 +81,40 @@
   margin: 0px 10px;
 }
 
-/* The container */
-.container {
-  display: block;
-  position: relative;
-  padding-left: 35px;
-  margin-bottom: 12px;
-  cursor: pointer;
-  font-size: 22px;
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  user-select: none;
-}
 
-/* Hide the browser's default checkbox */
-.container input {
-  position: absolute;
-  opacity: 0;
-  cursor: pointer;
-  height: 0;
-  width: 0;
-}
-
-/* Create a custom checkbox */
-.checkmark {
-  position: absolute;
-  top: 0;
-  left: 0;
-  height: 25px;
-  width: 25px;
-  border: solid #4158c4 2px;
-  border-radius: 50%;
-}
-
-/* On mouse-over, add a light purple background color */
-.container:hover input ~ .checkmark {
-  background-color: #4158c433;
-}
-
-/* When the checkbox is checked, add a purple background */
-.container input:checked ~ .checkmark {
-  background-color: #4158c4;
-}
-
-/* Create the checkmark/indicator (hidden when not checked) */
-.checkmark:after {
-  content: "";
-  position: absolute;
-  display: none;
-}
-
-/* Show the checkmark when checked */
-.container input:checked ~ .checkmark:after {
-  display: block;
-}
-
-/* Style the checkmark/indicator */
-.container .checkmark:after {
-  left: 6px;
-  top: 1px;
-  width: 9px;
-  height: 15px;
-  border: solid white;
-  border-width: 0 2px 2px 0;
-  -webkit-transform: rotate(45deg);
-  -ms-transform: rotate(45deg);
-  transform: rotate(45deg);
-}
-
-#saveButton {
-  /* Auto Layout */
 
+main {
   display: flex;
-  flex-direction: row;
-  justify-content: center;
+  flex-direction: column;
   align-items: center;
-  padding: 0px 16px;
-
-  position: absolute;
-  left: 6.24%;
-  right: 83.15%;
-  top: 56.95%;
-  bottom: 32.45%;
-
-  /* Primary */
-
-  background: #4158c4;
-  border-radius: 4px;
-  border: none;
-  /* Button Text */
-
-  position: static;
-  height: 18px;
-  left: 20.13%;
-  right: 20.13%;
-  top: calc(50% - 18px / 2);
-
-  /* Button/Large */
+}
 
-  font-family: Roboto;
+h5 {
+  font-family: "Roboto";
   font-style: normal;
-  font-weight: 500;
-  font-size: 18px;
+  font-weight: bold;
+  font-size: 24px;
   line-height: 100%;
-  /* identical to box height, or 18px */
-
-  display: flex;
-  align-items: center;
-  text-align: center;
-  letter-spacing: 0.0125em;
-  text-transform: capitalize;
-
-  color: #ffffff;
-
-  /* Inside Auto Layout */
-
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  margin: 0px 10px;
+  /* identical to box height, or 24px */
+  
+  
+  color: rgba(0, 0, 0, 0.87);
+  
+}
+
+p {
+width: 92px;
```

Comment:
⛏️ What do you think about indenting these lines within `p`?

Reply:
definitely!